### PR TITLE
feat: improve CLI user experience

### PR DESCRIPTION
**Pull request by OriginCode**
- init: add prompts to tell user critical changes to the system
- init: confirm before initialization and calling `update()`
- ui: `Dialoguer` use default simple theme
- i18n: Update translations, wrap lines
- macros: add `print_block_with_fl!()` to print message blocks without
  prefix
- macros: process multi-line strings in `println_with_prefix_and_fl!()`

### CODE REVIEW

**Comment by OriginCode** [comment]
@NeoChen1024 @MingcongBai Could you help review the translations (.tfl files)? Thank you so much!

File: i18n/zh-TW/systemd_boot_friend_rs.ftl [hunk 1]
```diff
@@ -6,25 +6,36 @@ invalid_esp = ESP_MOUNTPOINT 不正確
 invalid_index = 核心序號不正確
```
Comment (NeoChen1024) [inline i18n/zh-TW/systemd_boot_friend_rs.ftl]: 序號 -> 編號？

File: i18n/zh-CN/systemd_boot_friend_rs.ftl [hunk 2]
```diff
@@ -6,25 +6,36 @@ invalid_esp = ESP_MOUNTPOINT 不正确
...
+init = 正在安装并初始化 systemd-boot ...
+prompt_init =
+    systemd-boot-friend 即将安装及初始化 systemd-boot，并将其设置为默认 EFI 启动
+    项。完成后，您依旧可以在 EFI 启动管理器中访问其他已安装的启动引导器，如 GRUB
+    或 Windows 启动管理器。
+ask_init = 是否要初始化 systemd-boot-friend？
+update = 正在更新启动项 ...
+prompt_update =
+    systemd-boot 已成功初始化。是否要让 systemd-boot-friend 搜索 /boot 中的内核
+    并登记至 systemd-boot 配置中？若不需要，您可以随时执行
```
Comment (MingcongBai) [inline i18n/zh-CN/systemd_boot_friend_rs.ftl]: 并将其登记至 *

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: i18n/zh-TW/systemd_boot_friend_rs.ftl [hunk 3]
```diff
@@ -6,25 +6,36 @@ invalid_esp = ESP_MOUNTPOINT 不正確
...
 set_default = 正在將 {$kernel} 設為默認開機選項...
 ask_set_default = 是否要將 {$kernel} 設為默認開機選項？
 remove_default = 正在刪除默認開機選項 {$kernel} ...
 select_install = 請選擇要安裝的核心
 select_remove = 請選擇要刪除的核心
-ask_update = 是否要安裝所有核心及開機選項？（可在初始化完成後手動操作）
\ No newline at end of file
+init = 正在安裝並初始化 systemd-boot ...
+prompt_init =
+    systemd-boot-friend 即將安裝及初始化 systemd-boot，並將其設定為默認 EFI 開機
```
Comment (NeoChen1024) [inline i18n/zh-TW/systemd_boot_friend_rs.ftl]: 默認 -> 預設

File: i18n/zh-TW/systemd_boot_friend_rs.ftl [hunk 4]
```diff
@@ -6,25 +6,36 @@ invalid_esp = ESP_MOUNTPOINT 不正確
...
+    選項。完成後，您依舊可以在 EFI 開機管理程式中存取到其他已安裝的開機載入器，
```
Comment (NeoChen1024) [inline i18n/zh-TW/systemd_boot_friend_rs.ftl]: 存取到 -> 存取

**Pull request merged**